devkit/samples/ambiguous/ambiguous.feature

```diff
@@ -0,0 +1,6 @@
+Feature: Ambiguous steps
+  Multiple step definitions that match a pickle step result in an AMBIGUOUS status, since Cucumnber cannot determine
```

Comment:
For this can we include the expected steps so we know for the implementors what to write

Reply:
That's provided by the sample code in typescript I'd think.

Reply:
The issue with that is I cannot parse those from the gem / gem open command (They aren't contained in the package). I'd need to dig them up in the repo proper.

I'm not against that, but given that these steps almost never change. I'd prefer us to make it easier for writers in the bit of code that is copied down in the devkit sample copy

Reply:
We don't do that for any of the other samples either. And the exact step definitions and regular expressions are also in the message protocol.

I'd prefer not to duplicate information this way.